Source/JavaScriptCore/llint/LLIntOfflineAsmConfig.h

```diff
@@ -38,14 +38,16 @@
 #endif
 
 #ifdef __CHERI_PURE_CAPABILITY__
-#if __UINTCAP_WIDTH__ == 128
+#ifndef __SIZEOF_UINTCAP__
+#error __SIZEOF_UINTCAP__ is undefined
+#elif __SIZEOF_UINTCAP__ == 8
```

Comment:
These are too small

Reply:
Sorry, I missed that, force-pushed back to the old state.

Reply:
So they are! Thanks for spotting that. I'll fix it now.